### ISSUE TITLE
0 reading when offline (night)

### ISSUE THREAD

**Issue by ploman1973** [StephanJoubert/home_assistant_solarman#35]
During the downtime of my Sofar inverter, this integration output 0 (zero) as a total amount of kWhw produced. This makes the graphs detailed graph and the info in the Energy dashboard incorrect. Is there a way to keep the correct values when the inverter is offline?
![Schermafbeelding 2022-04-04 113343](https://user-images.githubusercontent.com/22667224/161524610-c79681a0-bcf3-4d58-a7a3-348ca482d5a1.png)

**Comment by pepelillo** [comment]
Yes, the inverter enter in standby mode at night, and it seems not to measure the CT grid total power (load comsumption) so the energy panel is not showing complete values. I think that there is a command to force the inverter not to enter in standby but in my case the electronics of the inverter draws a lot of power (when producing 4500w solar the inverter only gives 4300W) so I'm not sure that is a good idea, even there is some period at the end of the day that the inverter draws power from grid some time before enter in standby.
